test/666_abuse_of_start_test.sh

```diff
@@ -4,24 +4,38 @@
 
 start_suite "Abuse of 'start' operation"
 
-weave_on $HOST1 launch
+weave_on $HOST1 launch --log-level=debug
 docker_bridge_ip=$(weave_on $HOST1 docker-bridge-ip)
 proxy_start_container $HOST1 --name=c1
 
+check_hostconfig() {
+    docker_on $HOST1 attach c2 >/dev/null || true # Wait for container to exit
```

Comment:
on master this now suppresses errors with `>/dev/null 2>&1`. Should do here too.